site/content/docs/main/configuration.md

```diff
@@ -75,11 +75,11 @@ Where Contour settings can also be specified with command-line flags, the comman
 |---------------------------| ---------------------- |------------------------------------------------------------------------------------------------------| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | accesslog-format          | string                 | `envoy`                                                                                              | This key sets the global [access log format][2] for Envoy. Valid options are `envoy` or `json`.                                                                                                                                                                                       |
 | accesslog-format-string   | string                 | None                                                                                                 | If present, this specifies custom access log format for Envoy. See [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage) for more information about the syntax. This field only has effect if `accesslog-format` is `envoy` |
-| accesslog-level           | string                 | `info`                                                                                               | This field specifies the verbosity level of the access log. Valid options are `info`, `error` and `disabled`. |
+| accesslog-level           | string                 | `info`                                                                                               | This field specifies the verbosity level of the access log. Valid options are `info`, `error`, `critical` and `disabled`. |
```

Comment:
Now that there are several log levels for error responses, it could be good to mention the difference here in this doc page too: 

`error` (all non-success requests), `critical` (all 5xx requests)